docs/en/integrations/data-ingestion/apache-spark/index.md

```diff
@@ -0,0 +1,306 @@
+---
+sidebar_label: Integrating Apache Spark with ClickHouse
+sidebar_position: 1
+slug: /en/integrations/apache-spark/
+description: Introduction to Apache Spark with ClickHouse
+---
+
+# Integrating Apache Spark with ClickHouse
+
+[Apache Spark](https://spark.apache.org/) Apache Spark™ is a multi-language engine for executing data engineering, data science, and machine learning on single-node machines or clusters.
+
+# Get the Library
+
+## Download the Library
+
+The name pattern of binary jar is
+```
+clickhouse-spark-runtime-${spark_binary_version}_${scala_binary_version}-${version}.jar
+```
+you can find all available released jars under [Maven Central Repository](https://repo1.maven.org/maven2/com/github/housepower)
```

Comment:
are we going to update the Maven account later?

Reply:
Yes